modules/core/core-api/src/main/java/com/enonic/xp/app/ApplicationInstallationParams.java

```diff
@@ -11,7 +11,7 @@
     private ApplicationInstallationParams( final Builder builder )
     {
         start = builder.start;
-        triggerEvent = builder.triggerEvent;
+        triggerEvent = false;
```

Comment:
Do we need a field anymore? Wouldn't be better to return 'false' directly in getter?

Reply:
It still needs for `hashCode` and `equals` methods.